app/src/androidTest/java/com/duckduckgo/app/cta/ui/CtaViewModelTest.kt

```diff
@@ -695,6 +695,32 @@ class CtaViewModelTest {
         assertNull(fireDialogCta)
     }
 
+    @Test
+    fun whenRefreshCtaOnHomeTabAndReturningUsersNoOnboardingEnabledTrueAndWidgetCompatibleThenReturnNull() = coroutineRule.runBlocking {
```

Comment:
`AndWidgetCompatible` does this apply to this test?

I think for this test case, we want to highlight that user has not reached the threshold, wdyt?

Reply:
The part "ReturningUsersNoOnboardingEnabledTrue" means we're in the variant where the widget will never be shown, regardless of the threshold.

Reply:
gotcha, so `AndWidgetCompatible` means that in a normal scenario the widget would appear...

not sure if it's clear enough, but I don't have a better name...